runtime/strategies/map-slots.js

```diff
@@ -38,6 +38,11 @@ export default class MapSlots extends Strategy {
           return;
         }
 
+        // If there are any local slots, prefer them over remote slots.
```

Comment:
maybe just update `findAllSlotCandidates` to return {localSlots, externalSlots}, instead of filtering and reassigning?

Reply:
Done! Thanks!